### PR TITLE
Added dependency management

### PR DESCRIPTION
**Pull request by adamretter**

### CODE REVIEW

**Comment by joewiz** [comment]
@adamretter It looks like some conflicts have emerged. Could you update the PR please, perhaps after #41 and #15 are merged?

**Comment by adamretter** [comment]
@joewiz As requested I have rebased this. However, it probably makes sense these days to move it over to the Maven XAR build soon anyway, instead of using Ivy

File: Readme.md [hunk 1]
```diff
@@ -63,10 +63,30 @@ Each eXistdb installation to monitor is added as an instance entry at */db/apps/
 In the Monex Remote Monitoring tab click "Run" to start all remote monitoring jobs. You should now see an entry "localhost" beneath "Remote Monitoring" and beneath that an entry "Timelines".
 
 
+Building
+--------
+If you wish to build Monex from source code you should follow these steps:
 
- 
+1. Clone it from *wolfgangmm* (or from your own fork) `git clone https://github.com/wolfgangmm/monex.git`.
```
Comment (joewiz) [inline Readme.md]: Sorry, should now be `https://github.com/eXide/monex.git`

File: ivy.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<ivy-module version="2.0">
+    <info organisation="com.github.wolfgangmm" module="monex"/>
```
Comment (joewiz) [inline ivy.xml]: I presume the @organisation value should now be `com.github.exist-db` too.

File: Readme.md (hunk 1 above)
Comment (adamretter) [reply Readme.md]: Ah right. I have fixed that now.

File: ivy.xml (hunk 2 above)
Comment (adamretter) [reply ivy.xml]: Ah right. I have fixed that now.

File: build.xml [hunk 3]
```diff
@@ -26,16 +34,33 @@
...
     </path>
 
+    <available file="${ivy.lib}/${ivy.jar}" property="ivy.exists"/>
+    
+    <target name="bootstrap" description="Install ivy" unless="ivy.exists">
+        <mkdir dir="${ivy.lib}"/>
+        <get dest="${ivy.lib}/${ivy.jar}" src="http://search.maven.org/remotecontent?filepath=org/apache/ivy/ivy/${ivy.version}/${ivy.jar}"/>
+    </target>
+    
+    <target name="resolve" description="Use ivy to resolve classpaths" depends="bootstrap">
```
Comment (joewiz) [inline build.xml]: Ant returns:

> BUILD FAILED build.xml:46: The prefix "ivy" for element "ivy:resolve" is not bound.

I'd suggest adding `xmlns:ivy="antlib:org.apache.ivy.ant"` to the root `<project>` element or to the `<target>` that invokes the ivy tasks.

Comment (adamretter) [reply build.xml]: Okay here goes again...

**Comment by joewiz** [comment]
@adamretter Thanks! I've tested and confirmed that monex with this PR builds, installs, and works as normal. I agree that this would be a good one to mavenize!

**Pull request merged**